addons/mail/static/src/widgets/form_renderer/form_renderer_tests.js

```diff
@@ -49,6 +51,195 @@ QUnit.module('form_renderer_tests.js', {
     },
 });
 
+QUnit.test('[technical] messaging not created', async function (assert) {
```

Comment:
suggestion
```suggestion
QUnit.test('[technical] chatter displaying spinner while messaging not created', async function (assert) {
```
Looks functional enough to not be technical.